libs/SalesforceSDK/AndroidManifest.xml

```diff
@@ -79,6 +79,12 @@
             android:excludeFromRecents="true"
             android:theme="@style/SalesforceSDK.ActionBarTheme" />
 
+        <!-- IDP webview activity -->
+        <activity android:name="com.salesforce.androidsdk.auth.idp.IDPWebViewActivity"
```

Comment:
New activity for the `WebView` portion of the IDP flow. This is also the central orchestrator of this flow.

Reply:
Should we name it `IDPLoginActivity` instead to describe what it does rather than what it uses (and it's possible that in the future it won't use a web view anymore)?

Reply:
I deliberately named it this way. There will be another `IDPLoginActivity` that will be a subclass of `LoginActivity` in the next PR. That will be the screen that does login for a nonexistent user without switching users.

Reply:
Couldn't we use one activity for both?
If we need two separate  activities, could we still pick a name that describes what it does?

Reply:
Well....maybe I could. I'll only know when I implement the flow part :-)

I hesitate to call it `IDPLoginActivity` because that sounds like it's the IDP getting logged in. This generates a code for the SP though. How about `IDPCodeGeneratorActivity`?

Reply:
👍